monitoring/healthz/src/useServiceProviders.ts

```diff
@@ -83,14 +63,3 @@ export function hostSort(sps: SP[]) {
     new URL(sp.endpoint).hostname.split('.').reverse().join('.')
   sps.sort((a, b) => (hostSortKey(a) < hostSortKey(b) ? -1 : 1))
 }
-
-function unregisteredStageNodes() {
-  return [
-    {
-      delegateOwnerWallet: '0xb1C931A9ac123866372CEbb6bbAF50FfD18dd5DF',
-      endpoint: 'https://discoveryprovider4.staging.audius.co',
-      isRegistered: false,
-      type: { id: 'discovery-node' },
-    },
-  ]
-}
```

Comment:
don't think this was really used, shouldn't have unregistered nodes on healthz now that it's fully public probably